gatsby/content/projects/bridges/matrix-rss-bridge.mdx

```diff
@@ -0,0 +1,53 @@
+---
+layout: project
+title: matrix-rss-bridge
+categories:
+ - bot
+ - bridge
```

Comment:
```suggestion
```

This seem to be more of a bot than a bridge

Reply:
It is both a bot and a bridge, similar to feedbot which is categorized as both.

Reply:
Ah good catch, this is a mistake: at first sight feedbot is not a bridge either. The major difference between a bridge and a bot is that bridges a bi-directional.

I might have missed it, but I don't think this is the case for this project, is it?

Reply:
I suggest to remove the bridge section for now, and to add it in the future if you add features to make it bi-directional. Are you fine with me applying the suggestions and merging the PR?

Reply:
Yes